### ISSUE TITLE
Navigation buttons in Your Account lack sufficient color contrast

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1073]
Violation: Elements must have sufficient color contrast. Currently at color contrast of 4.14. Needs to meet 4.5.
